go/chat/convsource.go

```diff
@@ -56,15 +56,17 @@ func (s *baseConversationSource) SetTLFInfoSource(tlfInfoSource types.TLFInfoSou
 
 func (s *baseConversationSource) postProcessThread(ctx context.Context, uid gregor1.UID,
 	convID chat1.ConversationID, thread *chat1.ThreadView, q *chat1.GetThreadQuery,
-	finalizeInfo *chat1.ConversationFinalizeInfo) (err error) {
+	finalizeInfo *chat1.ConversationFinalizeInfo, checkPrev bool) (err error) {
```

Comment:
Minor preference for some named flag type (`type CheckPrevFlag int` or something) over the bool?